plugins/reach/lib/kReachUtils.php

```diff
@@ -180,24 +180,40 @@ public static function checkPriceAddon($entryVendorTask, $taskPriceDiff)
 		return $remainingCredit >= 0 ? true : false;
 	}
 	
-	public static function isDuplicateTask($entryId, $catalogItemId, $partnerId, $version, $allowResubmission)
+	public static function isDuplicateTask($entryId, $catalogItemId, $partnerId, $version, $allowResubmission, $manualFlow)
```

Comment:
IsDuplicateTask, should work without any relation if allowResubmission or manualFlow.
The logic due to this params should be managed elsewhere